### DISCUSSION TITLE
View-template 38

### DISCUSSION THREAD

**Discussion by mgakt** [OP-TED/eForms-SDK#742] [Q&A] [answered]
Dear All,
 
This conversation may become an issue. Today, we received a Contract modification notice(subTypeId:38). The XML message contains information on the right path; however, the PDF or HTML does not show the following BTs. 
BT-70
BT-77
BT-736
BT-761,BT-76
BT-720
BT-150
BT-3202
BT-1451
BT-145
BT-151
If we rely on the visualization provided by the OP, the content-based lawfulness check becomes extremely hard. If we ignore this visualization and use a custom design for content-based law checks, we can face a severe lawfulness problem because the official TED publication will still use the deficient visualization; this lawful notice will be the one published in Tenders Electronic Daily. ( For clarification, the XML message is prevalidated with CVS, and the validation does not show any issue. )
 
Of course, notice 38 is not only problematic from this point, but I'm opening the chain with this example.

**Comment by YvesJo** [comment, marked as answer]
Hi,
Please get in touch directly with the helpdesk ted@publications.europa.eu and provide further relevant details (e.g. the SDK version)
